src/desktop/components/main_layout/templates/scripts.jade

```diff
@@ -122,7 +122,7 @@ if options.stripe
   script( type="text/javascript", src="https://js.stripe.com/v2/" )
 
 if options.stripev3
-  script(async, id="stripe-js", src="https://js.stripe.com/v3/")
```

Comment:
I updated this to not be `async` since we need it at load time for the pages that require stripe (and it's configurable here). Open to other suggestions, though!